tests/utility/v2/test_ca_response_builder.py

```diff
@@ -264,13 +264,6 @@ def test_get_total_vulnerabilities(self):
         count = response_obj.get_total_vulnerabilities()
         self.assertEqual(count, 0)
 
-    def test_get_severity_unknown_values(self):
```

Comment:
Logic is complicated, can you add a test case

Reply:
Yes, Sure. That's why I tried to explain it in comments. 
will be adding Tests soon